test/package-check-failure.yml

```diff
@@ -0,0 +1,8 @@
+- tasks:
+  - name: install ansible
+    yum: name=ansible state=latest
+
+  - name: install ansible-lint
+    pip: name=ansible-lint
+    args:
+      state: latest
```

Comment:
not sure why state: latest is a key of args.

If you want the full YAML form, and certainly it's preferred these days:

```
  - name: install ansible-lint
    pip:
      name: ansible-lint
      state: latest
```